### DISCUSSION TITLE
This repository has been archived by the owner. It is now read-only.

### DISCUSSION THREAD

**Discussion by Robbotto** [community/community#35100] [Other Feature Feedback, Questions, & Ideas]
What does this mean: This repository has been archived by the owner. It is now read-only.
and can anyone tell me if this program is still installable: https://github.com/bakape/shamichan.  My hosting company has tried to install it and can't, so don't know if there is a problem with the program or if it is my hosting company?

**Comment by airtower-luna** [comment]
It means the repository is read-only, and the repository owner has basically declared they're not going to work on it any more. So don't expect any support or bugfixes from them. Under the conditions of the license you can of course make your own fixes (or pay someone to), even develop your own version (in your own repository).

See also the documentation: https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories

**Comment by Anmol-Baranwal** [comment]
The following points will clear your doubts. If you have any further queries, feel free to ask.

- Once a repository is archived, you cannot add or remove collaborators or teams.

- Archiving a repository makes it read-only to everyone (including repository owners). This includes editing the repository, issues, pull requests, labels, milestones, projects, wiki, releases, commits, tags, branches, reactions, and comments. 

- No one can create new issues, pull requests, or comments on an archived repository, but you can still fork archived repositories allowing development to continue elsewhere for archived open source projects. To make changes in an archived repository, you must unarchive the repository first.

**Comment by Robbotto** [comment]
Thanks for that1

Kind regards,

Jaime De Aguiar


—————————

Disclaimer

This e-mail and any attached files are intended for the named addressee only. It contains information, which may be confidential and legally privileged and also protected by copyright law. Unless you are the named addressee (or authorised to receive for the addressee) you may not copy or use it, or disclose it to anyone else. If you received it in error please notify the sender immediately and then delete it from your system.

**Comment by jfullstackdev** [comment]
I'm seeing lists of developers, if you are still into it, maybe you can contact them and say, you will be customizing it for your company and maybe if you will agree with the price of that, other open source projects are doing that really

**Comment by purnasth** [comment]
Does this also imply that the contributions we just made won't all be counted equally? Will my 10 opened pull requests from that repository, for instance, still count toward my contribution total on github? One repository was recently archived, so my overall number of PRs on the Frogtoberfest from that repo is no longer counting those PRs. Please explain to me.
